src/central_agentic_agent.py

```diff
@@ -69,17 +70,27 @@ def initialize_tools(self):
         """
         Initialize the tools available to the agent.
         """
-        melding = self.melding_attributes['MELDING']
-        straatnaam = self.melding_attributes['STRAATNAAM']
-        huisnummer = self.melding_attributes['HUISNUMMER']
-        postcode = self.melding_attributes['POSTCODE']
+        melding = self.melding_attributes.get('MELDING', '')
+        straatnaam = self.melding_attributes.get('STRAATNAAM', '')
+        huisnummer = self.melding_attributes.get('HUISNUMMER', '')
+        postcode = self.melding_attributes.get('POSTCODE', '')
+        not_allowed_tools = []
 
         os.environ["TRANSFORMERS_CACHE"] = cfg.HUGGING_CACHE
         os.environ["HF_HOME"] = cfg.HUGGING_CACHE
 
         self.WasteCollectionTool = WasteCollectionTool(straatnaam, huisnummer, postcode)
         self.BGTTool = BGTTool(straatnaam, huisnummer, postcode)
         self.PolicyRetrieverTool = PolicyRetrieverTool(melding)
+        if self.melding_attributes.get('LICENSE_PLATE_NEEDED', True):
```

Comment:
Why does this return `True` if `LICENSE_PLATE_NEEDED` does not exist? You wouldn't need the license plate in that case right?